UM-USElab/gradecraft-development
# 3013 gradebook page

### Status
**READY**

### Description
This PR:
- [x] re-adds the comprehensive gradebook page for instructors
- [x] adds a boxshadow to the display of horizontally scrolling tables to nudge users to realize that they contain more content than is immediately apparent
- [x] removes the separate grade_index page for admin users
- [x] adds the content of the grade_index page (minus some controls) to the student dashboard page so that instructors can see each grade a student has gotten 



### Related PRs
We'll need to update the visibility state represented when #3095 is merged

### Migrations
NO

### Steps to Test or Reproduce
1. Notice the new "Gradebook" link under the Coursework heading in the instructor - confirm that it loads properly
2. Navigate to a student and observe the way you can see a table of their grades on the show page

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Instructor navigation, student show page

======================
Closes #3013
